packages/forms/docs/09-testing.md

```diff
@@ -286,6 +286,45 @@ test('comments section is enabled' function () {
 });
 ```
 
+### Wizard
+
+To call a wizard's next step, use the `nextFormWizardStep()` method:
+
+```php
+use function Pest\Livewire\livewire;
+
+it('moves to next wizard step', function () {
+    livewire(CreatePostUsingWizard::class)
+        ->nextFormWizardStep()
+        ->assertHasFormErrors(['title']);
+});
+```
+
+For the subsequent steps you can pass a specific step's index to `currentStep`, which starts from 0 as the first step:
+
+```php
+use function Pest\Livewire\livewire;
+
+it('moves to the wizards second step', function () {
+    livewire(CreatePostUsingWizard::class)
+        ->nextFormWizardStep(currentStep: 1)
```

Comment:
I might be wrong, but doesn't this example put you on the 3rd step, since the currentStep is 1 which is the 2nd step?

Reply:
Right, I will update the example to have a clearer test description.
That aside, shouldn't it be better to input the desired wizard step instead of the current's index?

Reply:
maybe rename it to goToStep or setStep?

Reply:
That would be nice, I am also thinking on adding assertions for the current step `->assertCurrentStep(2)` or so, but it would be necessary to add a property to track it since it is set only on frontend.
In a project I a currently working, we track it like this:

```php

public int $currentStep = 1;

// override this to get the current step
// it would need to track the previous step too to be more accurate
public function dispatchFormEvent(...$args): void
{
    parent::dispatchFormEvent(...$args);
    // saves the current step, but only if validation is successfull
    // since in this event it starts from 0 we add +1
    // it should also be the next step, so +2
    if ('wizard::nextStep' === $args[0]) {
        $this->currentStep = $args[2] +2;
    }
}

```

Reply:
I got some neat ideas for improving this, I am going to work on it tonight and update it up to tomorrow, please confirm if it is a good idea to add a current step property and its getter method to the form/wizard object, if so I would add it on the existing listener for wizard::nextStep

Reply:
I was actually surprised that you can't access the current step.